ggml/src/ggml-cuda/vendors/hip.h

```diff
@@ -25,6 +25,7 @@
 #define CU_MEM_LOCATION_TYPE_DEVICE hipMemLocationTypeDevice
 #define CU_MEM_ACCESS_FLAGS_PROT_READWRITE hipMemAccessFlagsProtReadWrite
 #define CU_CHECK(fn) {hipError_t err = fn; if(err != hipSuccess) { GGML_ABORT("HipVMM Failure: %s\n", hipGetErrorString(err)); }}
+#define __shfl_sync(mask, var, laneMask, width) __shfl(var, laneMask, width)
 #define __shfl_xor_sync(mask, var, laneMask, width) __shfl_xor(var, laneMask, width)
```

Comment:
The synchronous version of the shuffle operators has been supported since rocm 6.2, probably we should just add the ability to conditionally use them based on HIP_VERSION >= 60200000 even when this wont do anything for now, just so it do sent get forgotten in the future.